app/models/miq_server/environment_management.rb

```diff
@@ -29,7 +29,7 @@ def minimal_env_options
         options.shift # remove the "minimal" from the front of the array
 
         # Special case where Netbeans is handling the UI worker for debugging
-        options.collect { |o| o.downcase == "netbeans" ? %w(schedule reporting noui) : o }.flatten
+        options.collect { |o| o.downcase == "netbeans" ? %w(schedule reporting no_ui) : o }.flatten
```

Comment:
According to our docs and current setup, one Generic worker and one UI worker are set up with MIQ_SPARTAN; these config values are for adding more workers to those defaults. This exception to the rule is for those like me that don't want a UI worker (as is default). 

Are you proposing that MIQ_SPARTAN shouldn't start a UI worker by default, and you'd have to add it like any other non-default worker here?